app/src/main/java/com/example/calculator/mvp/model/CalculatorModel.kt

```diff
@@ -0,0 +1,71 @@
+package com.example.calculator.mvp.model
+
+import com.example.calculator.mvp.contract.CalculatorContract
+import com.example.calculator.util.Constants.DIV
+import com.example.calculator.util.Constants.EMPTY
+import com.example.calculator.util.Constants.MINUS
+import com.example.calculator.util.Constants.PLUS
+import com.example.calculator.util.Constants.TIMES
+import com.example.calculator.util.Constants.ZERO
+
+class CalculatorModel : CalculatorContract.Model {
+
+    private var firstOperand = EMPTY
+    private var secondOperand = EMPTY
+    private var operator = EMPTY
+
+    private fun isResultZero(): Boolean{
+        return (operator.isEmpty() && firstOperand == ZERO)
+    }
+
+    override fun onPressedNumber(number: String) {
+        if (isResultZero()) {
+            firstOperand = number
+        } else {
+            if (operator.isEmpty()) {
+                firstOperand = firstOperand.plus(number)
+            } else {
+                secondOperand = secondOperand.plus(number)
+            }
+        }
+    }
+
+    override fun onPressedOperation(operation: String) {
+        operator = operation
+    }
+
+    override fun getResult(): String = firstOperand + operator + secondOperand
```

Comment:
make this as kotlin-style pls 
`"$firstOperand $operator $secondOperand"`

Reply:
Changed!